homeassistant/components/shopping_list/__init__.py

```diff
@@ -116,6 +117,10 @@ async def incomplete_all_service(call):
         """Mark all items in the list as incomplete."""
         await data.async_update_list({"complete": False})
 
+    async def clear_completed_items_service(call):
+        """Clear all completed items from the list."""
+        await data.async_clear_completed()
```

Comment:
Could you add test coverage for this service?

Reply:
I've added test-coverage for this service